x-pack/plugin/src/test/resources/rest-api-spec/api/data_frame.put_data_frame_transform.json

```diff
@@ -0,0 +1,21 @@
+{
+  "data_frame.put_data_frame_transform": {
+    "documentation": "TODO",
+    "methods": [ "PUT" ],
+    "url": {
+      "path": "/_data_frame/transforms/{transform_id}",
+      "paths": [ "/_data_frame/transforms/{transform_id}", "/_data_frame/transforms" ],
```

Comment:
`"/_data_frame/transforms"` is not supported on the REST layer.  The ID has to be in the URL.